mathesar/api/serializers/links.py

```diff
@@ -82,6 +82,13 @@ def create(self, validated_data):
     }
     link_type = serializers.CharField(required=True)
 
+    def run_validation(self, data):
+        if referent_table := data.get('referent_table', None):
+            referent_table_name = Table.current_objects.get(id=referent_table).name
+            if referent_table_name.find('(') and referent_table_name.find(')') != -1:
+                raise InvalidReferentTableName(referent_table_name)
```

Comment:
It would be better to use the `field` parameter of the `InvalidTableName` Exception class to specify which field of the body is failing the validation instead of having a different Exception.